Visuals: Turn signals on screen when blinker is used

HazZelnutz opened the pull request:
Adds a flashing right / left turn indicator, when a blinker is used.
This should solve #773.
<img width="2188" height="1146" alt="Screenshot from 2025-09-29 08-06-46" src="https://github.com/user-attachments/assets/1a2ca1d0-fae3-4812-882d-16aa6672715c" />

## Summary by Sourcery

New Features:
- Add flashing left/right turn signal indicators to the HUD when blinkers are active

nayan8teen commented:
@HazZelnutz Couple of things from a quick review:

- The turn indicator does not blink/flash.
- It overlaps with the SCC-V/SCC-M indicators & doesn't look visually appealing (see below)
- This needs a toggle so that users can chose to have it enabled or disabled.

<img width="640" height="292" alt="image" src="https://github.com/user-attachments/assets/7069097b-f60d-4f01-910f-37c274d74cba" />

HazZelnutz commented:
@nayan8teen Thanks for the quick test!
Points two and three should be easily adressable.
For the first one I have to have a deeper look.
On which hardware did you run it?

nayan8teen commented:
> @nayan8teen Thanks for the quick test!
> 
> Points two and three should be easily adressable.
> 
> For the first one I have to have a deeper look.
> 
> On which hardware did you run it?

On PC with replay. It displays the turn indicators as expected, but no blinking.

HazZelnutz commented:
@nayan8teen I modified the turn signals position and size to not conflict with the SCC-V/M indicators.
Also I added a switch to the visuals panel to enable or disable the feature.
Sadly I was not able to recreate your issue with the non-blinking tough.
Perhaps you could send me your test route so that I can give it a shot?

https://github.com/user-attachments/assets/f5e9758d-c255-468d-93af-8a212bda768c

nayan8teen reviewed (changes requested):
Looks pretty good!!
Made a few minor suggestions.

In addition, can you also draw them in a red color when lane change is blocked? 

Also, move them slightly lower to have them vertically aligned with the mph/kph text maybe.
